package.json

```diff
@@ -15,6 +15,7 @@
     "start": "node src/app.js",
     "lint": "eslint .",
     "test": "istanbul test -- _mocha",
+    "test-oracle": "LEDGER_DB_ENV=oracledev DYLD_LIBRARY_PATH=/opt/oracle/instantclient node node_modules/.bin/istanbul test -- _mocha",
```

Comment:
Thats what I did: `"test-oracle": "LEDGER_DB_ENV=oracledev DYLD_LIBRARY_PATH=/opt/oracle/instantclient istanbul test -- _mocha"`, but it doesn't seem to work. I'll investigate why.